apps/web/src/state/farmsV4/state/farmPools/fetcher.ts

```diff
@@ -64,7 +64,9 @@ export const fetchFarmPools = async (
   } catch (error) {
     console.error('Failed to fetch remote pools', error)
   }
-  const localPools = UNIVERSAL_FARMS.filter((farm) => {
+
+  const fetchFarmConfig = await fetchAllUniversalFarms()
```

Comment:
Do you think we can fetch these in parallel: 
`remotePools = await fetchExplorerFarmPools(args, signal)` and 
`const fetchFarmConfig = await fetchAllUniversalFarms()`

and handle error individually for remotePools?